### PR TITLE
adds guard around empty splintered.

### LINKED ISSUE #41: cover.js - TypeError: Reduce of empty array with no initial value
/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/contrib/cover.js:519
    splintered.reduce(function(p, c) {
               ^
TypeError: Reduce of empty array with no initial value
    at Array.reduce (native)
    at getSegments (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/contrib/cover.js:519:16)
    at /Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/contrib/cover.js:828:28
    at Array.forEach (native)
    at /Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/contrib/cover.js:811:14
    at Array.forEach (native)
    at CoverageSession.allStats (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/contrib/cover.js:796:31)
    at Transform._flush (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/index.js:76:31)
    at Transform.<anonymous> (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:135:12)
    at Transform.g (events.js:199:16)
    at Transform.emit (events.js:129:20)
    at finishMaybe (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:371:12)
    at endWritable (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:378:3)
    at Transform.Writable.end (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-coverage/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:356:5)
    at DestroyableTransform.onend (/Users/bbigelow/work/DAW/scriptsure_platform/node_modules/gulp-jasmine/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:523:10)
    at DestroyableTransform.g (events.js:199:16)

### PR DESCRIPTION
**Pull request by Jhorlin**
adds guard around empty splintered since it will throw and error when reducing.
fixes #41

### CODE REVIEW

File: contrib/cover.js [hunk 1]
```diff
@@ -516,6 +516,9 @@ function getSegments(code, lines, count, statementDetails) {
         return (a.start - b.start);
     });
     var combined = [splintered[0]];
+    if(splintered.length === 0){
```
Comment (dylanb) [inline contrib/cover.js]: Shouldn't this be moved above the line above?

**Comment by Jhorlin** [comment]
@dylanb I moved the splintered empty array guard up.

**Comment by Jhorlin** [comment]
@dylanb have you had a chance to review this pull request. I am working on a project and would love to remove my fork. Thanks

**Comment by dylanb** [comment]
Thanks for the contribution.

**Pull request merged**